lua/weapons/gmod_tool/stools/wire_debugger.lua

```diff
@@ -33,6 +33,46 @@ local function IsWire(entity) --try to find out if the entity is wire
 	return false
 end
 
+properties.Add("wire_debugger", {
+	MenuLabel = "Debug",
```

Comment:
This doesn't given any indication that clicking it again will stop debugging. This property would be nicer if it was a toggle property, or two mutually exclusive properties like `collision_on` / `collision_off`.